### DISCUSSION TITLE
Passwort generation, User And PW copy

### DISCUSSION THREAD

**Discussion by fabo123456** [lockerpm/.github#16] [Ideas]
Hi 
I would like to see some features

1. Password generation (in all apps)
I usually use passwords with 25 characters. Unfortunately, this always has to be redefined. It would be nice if the last setting was saved

2. copy ( Desktop)
If I want to copy the PW or user, I always have to click on the very small icon. It would be nicer if I could simply click on the corresponding line to copy an entry

Thanks

**Comment by luongnh261** [comment]
Hello @fabo123456 , thank you for raising the requests. The password generation has already been our product backlog, and we will also add the request for a more intuitive copy function on the desktop app to the backlog.
